stairway/src/main/java/bio/terra/stairway/FlightFilter.java

```diff
@@ -233,7 +354,96 @@ public String getParameterName() {
     public enum Datatype {
       STRING,
       TIMESTAMP,
+      LIST,
       NULL
     }
+
+    public static class Builder implements FlightFilterPredicateInterface.Builder {
```

Comment:
Everything in here is public. I wonder if it needs to be.

Reply:
I moved away from a builder so this should be a moot point now